stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/Messages.kt

```diff
@@ -105,17 +107,20 @@ public fun Messages(
             }
         }
 
-        val firstVisibleItemIndex = currentListState.firstVisibleItemIndex
-
         when {
+            newMessageState == Other && firstVisibleItemIndex < 3 -> {
```

Comment:
The `animateScrollToItem(0)` is called several times. Could you check with if something can be done with that? For example, we can check for `!currentListState.isScrollInProgress`.
```
newMessageState == Other && firstVisibleItemIndex < 3 !currentListState.isScrollInProgress ->
```

Reply:
That would be great - thanks for reminding me of this! :]

We can certainly test this, see how it behaves with a high load of messages.